javascript/javascript1/week3/smartphone-usage-app.js

```diff
@@ -0,0 +1,66 @@
+//Adding an activity
+const activities =[]
+
+function addActivity(activity, duration){
+  const today = new Date();
+  const date = today.toLocaleDateString("en-US");
+  activities.push({date:date, activity:activity, duration:duration })
+}
+addActivity( "YouTube", 60);
+addActivity("Instagram", 30);
+
+
+console.log(activities);
+
+//Show status and usage limit
+const quantity = activities.length;
+const limitation = 180;
+let totalDuration = 0 ;
+
+function showStatus(){
+  if (quantity !== 0){
+    for(let i = 0; i < activities.length; i++){
+      totalDuration += activities[i].duration; 
+    }
+    if (totalDuration< limitation)
+      return `You have added ${quantity} activities. They amount to ${totalDuration} min. of usage.`
+    else
+      return `You have added ${quantity} activities. They amount to ${totalDuration} min. of usage.
+You have reached your limit, no more smartphoning for you!`
+  } else{
+    return `Add some activities before calling showStatus.`;
+  } 
+}
+
+//Test cases
+let myStatus = showStatus();
+console.log(myStatus);
+
+addActivity("Call of Duty", 90);
```

Comment:
You add 90 minutes of activity here, but it seems your `console.log` says that the total amount of activities have remained unchanged and that the total have increased by 180 minutes and not 90. Can you see what goes wrong? 🐛 

```
You have added 2 activities. They amount to 90 min. of usage.
You have added 2 activities. They amount to 270 min. of usage.
```

(Hint 1: it is two separate issues.) (Hint 2: Avoid globals! 🌐)